src/test/rules/handleCallbackErrRuleTests.ts

```diff
@@ -86,7 +91,35 @@ ruleTester.addTestGroup('custom-error-name-pass', 'should pass with custom error
 ]);
 
 ruleTester.addTestGroup('custom-error-name-fail', 'should fail with custom error name', [
-  { code: `function(errorMsg, stream) { }`, options: ['errorMsg'], errors: [error] }
+  { code: `function(errorMsg, stream) { }`, options: ['errorMsg'], errors: [error] },
+  {
+    code: `error => console.error('Could not print the document');`,
+    options: ['error'],
+    errors: [error]
+  },
+  {
+    code: `error => something.something.error.something.err('Could not print the document');`,
+    options: ['^(err|error)$'],
+    errors: [error]
+  },
+  {
+    code: `error => console.error(error.stack);`,
```

Comment:
It passes, notice that this test does not expect any `errors` like the next one.

```
{
     code: `error => console.error(error.stack);`,
     options: ['^(err|error)$', { allowProperties: false }],
     errors: [strictError]
   },
```

The one expects the strict error since the allow properties is turned off.